.github/workflows/performancetest.yml

```diff
@@ -0,0 +1,40 @@
+name: Performance-Test
+
+on:  
+  push:
+    branches:
+      - development
+
+jobs:
+  slsart-perf:
+    name: slsart Performance Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
+    env:
+      NPM_CONFIG_PREFIX: "~/.npm-global"
+      STAGE_NAME: merge-perf-test
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Did you already ask @toddysm do add these secrets to the development branch? I'm assuming that needs to happen before merge.

Reply:
Yeah, we're going to work on that tomorrow.